packages/utils/tsconfig/client.json

```diff
@@ -1,10 +1,19 @@
 {
   "$schema": "http://json.schemastore.org/tsconfig",
-  "extends": "./base.json",
   "compilerOptions": {
+    "target": "ESNext",
     "module": "ESNext",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "jsx": "react-jsx",
-    "noEmit": true
+    "moduleResolution": "Node",
+    "useDefineForClassFields": true,
+    "lib": ["DOM", "DOM.Iterable", "ESNext"],
+    "allowJs": false,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
previously it was loading the base which itself used node 16 and had it, I separated the 2 to avoid conflicts but had to readd as the helper plugin is still doing import x from instead of import * as x from for some packages.